apps/web/src/app/(default)/webkom/page.tsx

```diff
@@ -78,6 +78,7 @@ export default function WebkomPage() {
         <UnorderedList className="list-none p-0 text-lg">
           <ListItem>Andreas Drevdal</ListItem>
           <ListItem>Birk Monsen</ListItem>
+          <ListItem>Birk Monsen</ListItem>
```

Comment:
Tror navnet ditt kommer opp to ganger her

Reply:
Ellers ser det veldig bra ut!

Reply:
Tror eg bare legger ned pullrequesten og lager en ny for denne filen skal jo isje være med ;-;